.github/workflows/documentation.yml

```
@@ -38,7 +41,7 @@ jobs:
       - name: Check out project
         if: github.event_name == 'workflow_call'
         uses: actions/checkout@v4
-
+e
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)